### PR TITLE
Roywill main

### PR DESCRIPTION
**Pull request by roywill**
Updated with suggestions

### CODE REVIEW

File: draft-ietf-scitt-architecture.md [hunk 1]
```diff
@@ -974,15 +974,12 @@ All contents exchanged between actors is protected using secure authenticated ch
 
 #### Signed Statements and Their Registration
 
-The Transparency Service is trusted with the confidentiality of the Signed Statements presented for Registration.
-Some Transparency Services may publish every Transparent Statement in their logs, to facilitate their dissemination and auditing.
-Others may just return Receipts to clients that present Singed Statements for Registration, and disclose the Append-only Log only to Auditors trusted with the confidentiality of its contents.
+The Transparency Service is trusted with the confidentiality of the Signed Statements presented for registration. The act of registration will always record in the
+logs the Signed Statement.  There is a use case where the logs are completely publicly accessible, and another where the audit requires elevated permission (this could be for privacy or for mitigation of servicing expensive operations). In either case the returned Receipts to the client will both facilitate simple evaluation of meeting the identity policy bar and act as a key to allow query during the elevated deep audit.
```
Comment (yogeshbdeshpande) [inline draft-ietf-scitt-architecture.md:978]: Please note with your changes, we need to remove line : 980 below, as we are not registering Transparent Statements in the transparency service!

Comment (henkbirkholz) [reply draft-ietf-scitt-architecture.md:978]: remove or change to... see below?

File: draft-ietf-scitt-architecture.md [hunk 2]
```diff
@@ -974,15 +974,12 @@ All contents exchanged between actors is protected using secure authenticated ch
...
 
 A collection of Transparent Statements must not leak information about the contents of other Transparent Statements registered on the Transparency Service.
```
Comment (henkbirkholz) [inline draft-ietf-scitt-architecture.md:980]:
```suggestion
A collection of Transparent Statements must not leak information about the contents of other Signed Statements registered on the Transparency Service.
```

**Review by henkbirkholz — CHANGES_REQUESTED** [review]

**Review by OR13 — CHANGES_REQUESTED** [review]
please apply the suggestion from @henkbirkholz

**Comment by SteveLasker** [comment]
@roywill, can you please address the changes above?

**Review by henkbirkholz — APPROVED** [review]
lgtm, any objections?

**Review by OR13 — APPROVED** [review]

**Comment by OR13** [comment]
iirc, this PR is stale, has already been implemented by other PRs, and has conflicts... merge it or close it...

**Comment by yogeshbdeshpande** [comment]
I agree with @OR13 fully. This has been already addressed by other PR's so we can close this!

**Comment by SteveLasker** [comment]
closing per comments. 
@roywill please reopen if needed.

**Comment by roywill** [comment]
I will be on it tomorrow.

Roy Williams
________________________________

**Pull request closed without merging**